[EDU-1674] - Add pub/sub message encryption example (#2361)
ably/docs: examples/pub-sub-message-encryption/page.md

# Sending and receiving encrypted messages with Pub/Sub

> Enable clients to send and receive encrypted messages in a channel.

Reviewer: ```suggestion
Enable clients to send and receive encrypted messages on a channel.
```

> Messaging enables instant, scalable communication through publishing and subscribing to channels. Messages represent dynamic data payloads that clients publish and subscribe to, powering collaborative applications across realtime newsfeeds, auction applications or financial websites.

Reviewer: ```suggestion
Messaging enables instant, scalable communication through publishing and subscribing to channels. Messages represent data payloads that clients publish and subscribe to. They are the basis of realtime communication in all applications.
```

> Encryption is enabled by default in the Ably SDKs ensuring data is sent and received securely. Messages however are not encrypted and cannot be decrypted by Ably and are encrypted on a channel by channel basis.

Reviewer: ```suggestion
TLS (Transport Layer Security) is enabled by default in Ably SDKs to ensure data is sent and received securely. However, messages are not encrypted by default. Encryption can be configured on a channel-by-channel basis to make message payloads opaque, so that they can only be decrypted by other clients sharing a secret key.
```

> Messaging and encryption is implemented using [Ably Pub/Sub](https://ably.com/docs/auth). The Pub/Sub SDK provides a set of flexible APIs capable of building any realtime application and is powered by Ably's reliable and scalable platform.

Reviewer: ```suggestion
Message encryption is implemented using [Ably Pub/Sub](https://ably.com/docs/auth). The Pub/Sub SDK provides a set of flexible APIs capable of building any realtime application and is powered by Ably's reliable and scalable platform.
```
